src/components/common/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+type Props = {
+  variant: "Sign in" | "Sign up";
+};
+
+const Button = ({ variant }: Props) => {
```

Comment:
```suggestion
const Button = ({ entry }: Props) => {
```
`variant`は抽象的すぎてどのコンポーネントのpropsにも当てはまる気がするのでentryとかの方がより具体的で分かりやすいかもです